src/MapViewController.swift

```diff
@@ -89,7 +215,8 @@ open class MapViewController: UIViewController, LocationManagerDelegate {
       return tgViewController.cameraType
     }
   }
-  
+
+  /// The current position of the map in latitude / longitude.
```

Comment:
TGGeoPoint uses `longitude, latitude`. https://github.com/tangrams/tangram-es/blob/master/platforms/ios/src/TangramMap/TGGeoPoint.h, we can probably reverse those to avoid confusion in the order, even if they are explicitly named.